### PR TITLE
[RocksDBStore]Add some arguments related to read-only

### PR DESCRIPTION
**Pull request by boscohyun**
SSIA

### CODE REVIEW

**Comment by boscohyun** [comment]
Is there any guide for writing a change log?

File: CHANGES.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ To be released.
 [#2520]: https://github.com/planetarium/libplanet/pull/2520
 [#2529]: https://github.com/planetarium/libplanet/pull/2529
 
+### Others
+
+ - (Libplanet.RocksDBStore) Add some arguments to `RocksDBUtils.OpenRocksDb()` to be possible to open it read-only mode. [#2535]
```
Comment (limebell) [inline CHANGES.md:52]:
```suggestion
 -  (Libplanet.RocksDBStore) Add some arguments to `RocksDBUtils.OpenRocksDb()` to be possible to open it read-only mode. [#2535]
```
It's minor changelog convention. :)

Comment (boscohyun) [reply CHANGES.md:52]: @limebell Thank you!

**Comment by boscohyun** [comment]
I did not search before make this pull request.😅 There is more efficient [pull request](https://github.com/planetarium/libplanet/pull/2470).

**Pull request closed without merging**